tests/test_validators.py

```diff
@@ -129,6 +133,26 @@ def test_html_snippet_invalid_urls(self):
             self.all_urls_form.process(content_with_urls=snippet)
             self.assertEqual(self.all_urls_form.validate(), False)
 
+    def test_required_if(self):
+        self.required_if_form.process(content=u"Content", blurb=u"Blurb")
+        self.assertTrue(self.required_if_form.validate())
```

Comment:
We're using Pytest's `assert` statement in all new tests. Older tests can be migrated gradually.